docs/installation.rst

```diff
@@ -238,21 +243,21 @@ Alternatively, you can enumerate all build dependencies in the command line::
     python3-sphinx_rtd_theme gcc gcc-c++
     sudo zypper install python3-llfuse  # llfuse
 
-Mac OS X
+macOS
 ++++++++
 
 When installing via Homebrew_, dependencies are installed automatically. To install
 dependencies manually::
 
-    brew install python3 openssl
+    brew install python3 readline openssl zstd lz4 xz xxhash
```

Comment:
readline and xz are usually needed for building python, but not for borg!?

Reply:
borg supports lzma ("xz") compression, but it does that by using it via Python stdlib.

Reply:
Right. I've confirmed that Borg from Github builds without `readline` and `xz`. So removed them.